base-containers/base/bin/_run_student_intern

```diff
@@ -21,6 +21,7 @@ logger = setup_logger()
 # Check the runtimes
 runtime = sys.argv[1]
 parent_runtime = sys.argv[2]
+kvm_enabled = sys.argv[3] == "True"
```

Comment:
I'd document the args at both sides, here and at the call side in `_docker_interface.py` (to avoid implementing a useless argumentparser that would be self documenting)

Reply:
The arguments managment will be replaced by an argparser.